### PR TITLE
get flood server onto docker hub, with fixes

### PR DESCRIPTION
**Pull request by trevj**
I wanted to get our flood server image onto Docker Hub and fell into a - useful, admittedly - rabbit hole.

TL;DR: because `dd if=/dev/zero count=1 bs=100M` takes **vastly** longer to run than `dd if=/dev/zero count=1 bs=100M > /dev/null` we haven't always been testing our proxy at its limits

Summary of changes:
 * do not build flood server on-demand
 * remove the rate limiting stuff (`pv`) because it's been a long time since we didn't implement back-pressure
 * consume the pipe as fast as possible

I ran a bunch of benchmarks: Chrome, Firefox, Node.js, with and without obfuscation, with and without latency between peers, etc. etc.

What surprised me:
 * Chrome is a beast. Unobfuscated, my system - a 3.5Ghz Xeon E5 - can push 15MB/sec through Zork. **That's 3x Firefox and 10x Node.js.**
 * Obfuscation changes this: **Firefox pushes 1.5x as much per second as Chrome** (1.8Mb/sec vs. 1.2Mb/sec)
 * Latency between peers affects Node.js very badly: at 150ms between peers, it only pushes 1/3 as much data per second as Chrome or Firefox. I wonder if this is the same poor buffer handling at the SCTP layer as old versions of Firefox?

Full results:

#### 0ms latency
chrome,14611
firefox,5781
node,1523

#### 150ms latency
chrome,604
firefox,612
node,225

#### 0ms latency
chrome,1289
firefox,1772
node,872

#### 150ms latency
chrome,535
firefox,532
node,192

#### 0ms latency
chrome,1208
firefox,1814
node,817

#### 150ms latency
chrome,529
firefox,530
node,195

### CODE REVIEW

**Comment by soycode** [comment]
:+1:, and interesting results. I agree that buffer handling in node may be an issue (especially because node buffers are somewhat different creatures than regular JavaScript buffers).

**Comment by trevj** [comment]
Ah! I hadn't even thought of that but good point!

The buffers I'm referring to are at the SCTP layer. Here is a summary of our work to make this work in Chrome 47 (not Firefox, as I originally said):
https://github.com/uProxy/uproxy/issues/1339

At the uProxy layer, this might be relevant too:
https://github.com/uProxy/uproxy/issues/1563
